docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx<3.5.0
 sphinx-rtd-theme==0.4.3
 pyrovision
+python-dotenv >= 0.15.0
```

Comment:
I didn't notice before : these are not a documentation requirements, both these should be moved to https://github.com/pyronear/pyro-engine/blob/master/requirements.txt

(I suggest replacing `pyrovision` with `pyrovision>=0.1.2` while you are at it)